src/main/java/srver/server.java

```diff
@@ -0,0 +1,181 @@
+package srver;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.*;
+import java.net.ServerSocket;
+import java.net.Socket;
+import java.net.URL;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
+
+public class server {
+
+    private static HashMap<String, Connection> connections;
+
+    public static void main(String[] args) throws IOException, JSONException, InterruptedException {
+        connections = new HashMap<String, Connection>();
+        Accept Acc;
+        Acc = new Accept();
+        Acc.start();
+        String s1;
```

Comment:
&#x1F34E; **Issue**
There is no reason to declare this field here. Put it 3 lines below when you initialize it.